### PR TITLE
Simona p

### PR DESCRIPTION
**Pull request by Simona888**
Please check my task solution for Array_MoveZerosToTheEnd :-)

### CODE REVIEW

**Review by emirayral — APPROVED** [review]
Great work!

**Review by pheonixzee — APPROVED** [review]
great

File: src/SimonaTasks/anagram_of_two_Strings/Anagram_task01.java [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+            }
+
+        } else {
+            System.err.println(str1 + " and " + str2 + " are NOT anagram");
+
+        }
+
+    }
+
+
```
Comment (AnnaChupina) [inline src/SimonaTasks/anagram_of_two_Strings/Anagram_task01.java:51]: Great job!

**Review by AnnaChupina — COMMENTED** [review]
Great job!

**Comment by KateKing29** [comment]
👍

**Review by AnnaChupina — COMMENTED** [review]
Great!

**Review by AnnaChupina — COMMENTED** [review]
Great!

**Pull request merged**